### PR TITLE
Remove countries not covered by GDPR

### LINKED ISSUE #764: Only allow GDPR countries in the schema
Our schema currently allows all countries to be set as `relevant-countries`:

https://github.com/datenanfragen/data/blob/baf04307c6d152da3c561a99eef6c49810f42a35/schema.json#L20

However, this doesn't really make sense as we only care about the GDPR. We should change the schema to only accept countries where the GDPR is applicable. This of course includes all EU countries but also some others.

[This post](https://beasleydirect.com/gdpr-countries/) might be a good starting point, although I am not sure if it is complete.

### PR DESCRIPTION
**Pull request by sudiptog81**
Kept only those ISO 3166-1 alpha-2 country codes for countries where GDPR is applicable.

Closes #764.

### CODE REVIEW

**Comment by sudiptog81** [comment]
There were some issues with the schema validation, I'll try to resolve those, squash the commits and then mark the PR for review.

File: schema.json [hunk 1]
```diff
@@ -17,7 +17,12 @@
 			"uniqueItems": true,
 			"items": {
 				"type": "string",
-				"enum": [ "all", "at", "be", "bg", "hr", "cy", "cz", "dk", "ee", "fi", "fr", "de", "gr", "hu", "ie", "it", "lv", "lt", "lu", "mt", "nl", "pl", "pt", "ro", "sk", "si", "es", "se", "uk", "gb", "no", "is", "li", "ic", "gp", "gf", "mq", "yt", "re", "mf"]
```
Comment (baltpeter) [inline schema.json]: Please keep this on one line, as previously.

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thank you, looks good! I have left just two small comments below.

File: companies/dreamsoftinfotech.json [hunk 2]
```diff
@@ -1,8 +1,6 @@
 {
     "slug": "dreamsoftinfotech",
-    "relevant-countries": [
-        "in"
-    ],
+    "relevant-countries": [],
```
[the remark this answers is not in the document]
Comment (sudiptog81) [reply companies/dreamsoftinfotech.json]: Addressing it in the next commit.

File: schema.json (hunk 1 above)
Comment (sudiptog81) [reply schema.json]: Addressed this in the next commit.

File: companies/dreamsoftinfotech.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (sudiptog81) [reply companies/dreamsoftinfotech.json]: Addressed this in the next commit.

**Comment by sudiptog81** [comment]
Thank you for reviewing my PR. I have tried to address the requested changes.

**Review by baltpeter — APPROVED** [review]
Great, thank you!

**Pull request merged**